marketplace/commercetools-products/src/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+  </head>
+  <body>
+    <div id="root"></div>
+    <script src="https://ct-merchant-test.herokuapp.com/app.bundle.js"></script>
```

Comment:
What's the reason behind having this external JS? Is this something we need for commerce tools to work?

Reply:
This looks shady: `herokuapp`, no version in the filename (can it break w/o us knowing?)

Reply:
This is the product picker provided by Commerce Tools. Since they wrote this because of this extension (though they'll use it for other integrations in the future) it's not deployed nicely (yet). I'll raise this with them before we publish it to the Marketplace.